File: php/app/LeagueManagement/Models/Player.php
<?php

namespace TrentonDarts\LeagueManagement\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\SoftDeletes;

class Player extends Model
{
    use SoftDeletes;
    /**
     * The database table used by the model.
     *
     * @var string
     */
    protected $table = 'players';

    /**
     * The attributes that are mass assignable.
     *
     * @var array
     */
    protected $fillable = ['leagueId',
        'userId',
        'firstName',
        'lastName',
        'nickname',
        'email',
        'homePhone',
        'cellPhone',
        'shirtSize',
        'address1',
        'address2',
        'city',
        'state',
        'zip',
        'acceptText',
        'acceptEmail'];

    public function getNameAttribute(){
        return $this->attributes["firstName"]." ".$this->attributes["lastName"];
    }
}

File: src/DartLeague/DartLeague.Repositories/LeagueData/Player.cs
using System;
using System.Collections.Generic;

namespace DartLeague.Repositories.LeagueData
{
    public partial class Player
    {
        public int Id { get; set; }
        public bool AcceptEmail { get; set; }
        public bool AcceptText { get; set; }
        public string Address1 { get; set; }
        public string Address2 { get; set; }
        public string CellPhone { get; set; }
        public string City { get; set; }
        public string Email { get; set; }
        public string FirstName { get; set; }
        public string HomePhone { get; set; }
        public string LastName { get; set; }
        public int LeagueId { get; set; }
        public string Nickname { get; set; }
        public string Notes { get; set; }
        public string ShirtSize { get; set; }
        public string State { get; set; }
        public int UserId { get; set; }
        public string Zip { get; set; }
    }
}
